From 7f83d7a4da245aed120f550da297f61cd00c2289 Mon Sep 17 00:00:00 2001
From: Ian Kent <raven@themaw.net>
Date: Thu, 4 Jun 2020 13:29:30 +0800
Subject: [PATCH 13/13] autofs-5.1.6 - fix double quoting of ampersand in
 auto.smb as well

The example program mount script installed to /etc/auto.smb incorrectly
adds a quote for the & character that causes mount failures. But the
produced map entry is already surrounded by double quotes. This may have
been handled by mount.cifs at some point but it's failing now.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 samples/auto.smb | 1 -
 1 file changed, 1 deletion(-)

diff --git a/samples/auto.smb b/samples/auto.smb
index 4842b68..f6d41d3 100755
--- a/samples/auto.smb
+++ b/samples/auto.smb
@@ -75,7 +75,6 @@ $SMBCLIENT $smbopts -gL "$key" 2>/dev/null| awk -v "key=$key" -v "opts=$opts" -F
 		  dir = $2
 		  loc = $2
 		  # Enclose mount dir and location in quotes
-		  gsub(/\&/,"\\\\&",loc)
 		  print " \\\n\t \"/" dir "\"", "\"://" key "/" loc "\""
 		}
 	END 	{ if (!first) print "\n"; else exit 1 }
-- 
2.27.0

